docs/installingNative.md

```diff
@@ -32,7 +32,7 @@ Most of these prerequisites can be installed using the package manager of your O
     ```
 
     ```sh
-    brew install git graphviz postgresql@15 redis@6.2
+    brew install git graphviz postgresql redis uv d2
```

Comment:
We need postgres >= 16. Installing a specific version with `brew install postgresql@16` means that we need to manually add a PATH to `.zshrc`, which is kind of ugly. I prefer to install the latest version (at the moment that's postgres 17) and just live with the fact that we are running a newer version locally.

We can also use `brew` to install `uv` and `d2`.

Reply:
I think the version should be pinned for postgresql, even if the PATHing is a pain.

Reply:
I agree with Matt here.

> Mainly because postgres in my understanding (I could be wrong) doesn't make a lot of guarantees of compatibility across versions.

For the data format on disk, sure, but that's hardly relevant here. For syntax/functionality used in queries, they basically always maintain backwards-compatibility. I also run the latest version of Postgres in development and don't experience issues. Of course, we should be careful that we don't use new syntax/features that isn't available in Postgres 16, but (if we happen to do so, and we haven't yet) that would be easily caught in testing or CI.